media/media_options.gni

```diff
@@ -57,6 +57,11 @@ declare_args() {
     media_use_libvpx = false
   }
 
+  # Chrobalt doesn't use ffmpeg.
+  if (is_cobalt) {
+    media_use_ffmpeg = false
+  }
```

Comment:
I had a similar need to force-define `enable_vulkan` to false and I was pointed to adding it as an args.gn overwrite in [cobalt/build/configs/linux-x64x11/args.gn](https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/main:cobalt/build/configs/linux-x64x11/args.gn;l=9-12?q=enable_vulkan&ss=h%2Flbshell-internal%2Fcobalt_src%2F%2B%2Frefs%2Fheads%2Fmain:cobalt%2F).

Could this overwrite also go there?

Reply:
@niranjanyardi @andrewsavage1 for opinions/keeping me real.

Reply:
+1 to this, @niranjanyardi let's try to get instructions up for this soon